### ISSUE TITLE
It seems there is only the last 30 days in the borgbase

### ISSUE THREAD

**Issue by kelson42** [kiwix/borg-backup#72]
This is not normal. We should:
* Have all the backups since two years (as we don't run prune)
* Ideally a smarter pruning

**Comment by rgaudin** [comment]
You are wrong. There currently are 25 archives in the download backup repo.

here are the dates:

- 31/12/2020
- 31/01/2021
- 28/02/2021
- 31/03/2021
- 30/04/2021
- 31/05/2021
- 30/06/2021
- 31/07/2021
- 31/08/2021
- 30/09/2021
- 31/10/2021
- 30/11/2021
- 12/12/2021
- 18/12/2021
- 26/12/2021
- 31/12/2021
- 02/01/2022
- 09/01/2022
- 13/01/2022
- 14/01/2022
- 15/01/2022
- 16/01/2022
- 17/01/2022
- 18/01/2022
- 19/01/2022

In summary, there are:
- 1 per day for the last 7 days
- 1 per week in the last 5 weeks (approx.)
- 1 per month for the last 12 months

![vorta](https://user-images.githubusercontent.com/57929/150339609-43219fbf-ef54-4daa-968e-420fb28c9952.png)

**Comment by rgaudin** [comment]
You are wrong. There currently are 26 archives in the download backup repo.

here are the dates:

- 31/12/2020
- 31/01/2021
- 28/02/2021
- 31/03/2021
- 30/04/2021
- 31/05/2021
- 30/06/2021
- 30/07/2021
- 31/08/2021
- 30/09/2021
- 31/10/2021
- 30/11/2021
- 12/12/2021
- 19/12/2021
- 26/12/2021
- 31/12/2021
- 02/01/2022
- 09/01/2022
- 13/01/2022
- 14/01/2022
- 15/01/2022
- 16/01/2022
- 17/01/2022
- 18/01/2022
- 19/01/2022
- 20/01/2022

In summary, there are:
- 1 per day for the last 8 days
- 1 per week in the last 6 weeks
- 1 per month for the last 13 months

![vorta](https://user-images.githubusercontent.com/57929/150341676-644bb282-9f15-4f1f-b904-5b9c50694c43.png)
